sale_order_weight/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Andrea Cometa - Apulia Software
```

Comment:
Please use Copyright instead of ©, see https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L2 as reference.